src/libraries/System.Runtime.InteropServices/tests/Ancillary.Interop/VirtualMethodIndexAttribute.cs

```diff
@@ -8,62 +8,63 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace System.Runtime.InteropServices.Marshalling
+namespace System.Runtime.InteropServices.Marshalling;
```

Comment:
(In this and other files) we'll probably need to switch away from file-scoped namespaces before we move this code to the `src` projects that actually ship. Should we just do that now for consistency (also reducing the diff size in this PR)?